### ISSUE TITLE
Pinch-Zoom Sensitivity - iPad

### ISSUE THREAD

**Issue by Jacob-Kizzee** [DDMAL/diva.js#454]
I'm experiencing hyper sensitivity with pinch-zooming on iPad (Safari).  This is also seen on multiple other mobile devices (android and iOS based, phone and tablet).  

This may be a repeat of #30 , but no solution code was posted.

Any zoom motion (up, down, pinch-expand) results in 2 to 3 levels of zoom (pinch-zoom increments by a decimal amount that is not exactly an integer).

**Q**:  Is there a sensitivity setting in the options or has anyone seen this issue since #30 ?

In the same thread, on my application the zoom during pinch seems to be jumping directly to each zoom level and not incrementing by a decimal like the default Diva example (the Diva example is still sensitive).  Maybe I'm missing a setting?

Thanks for your time!

**Comment by EricHanLiu** [comment]
I don't have access to a tablet and the Chrome debugging mobile emulator's pinch zoom feature doesn't work properly on my end. Using the gesture events don't actually change the zoom (but panning works properly).

On my phone, pinch zooming doesn't do anything either.

**Comment by Jacob-Kizzee** [comment]
After review I did the following to resolve the issue on my end:

With the Document-Handler.js file I moified the math for calculating the new ZoomLevel within the DocumentHandler.prototype.onPinch function.

The calculation for newZoomLevel goes from:
`var newZoomLevel = Math.log(Math.pow(2, settings.zoomLevel) * endDistance / (startDistance * Math.log(2))) / Math.log(2);`
To:
`var newZoomLevel = ((endDistance - startDistance) / Math.abs((startDistance - endDistance)) * settings.zoomLevels * settings.zoomTouchSensitivity) + settings.zoomLevel;`

Note the addition of a new setting 'settings.zoomTouchSensitivity'.  This sensitivity is paired with the zoomLevels in the calculation to produce a smooth zoom for mutliple detail levels.

I find a value of .015 for the sensitivity works well.

Issue is resolved and closed on my end.  Thanks!
